amnestywebsite/humanity-theme
# Update translations

Ref: https://github.com/amnestywebsite/amnesty-wp-theme/issues/2860

Related:
- https://github.com/amnestywebsite/wp-plugin-theme-companion/pull/49
- https://github.com/amnestywebsite/wp-plugin-theme-companion/pull/50
- https://github.com/amnestywebsite/wp-plugin-amnesty-branding/pull/13

Steps to test:
1. set your profile language to "site default"
2. visit the dashboard in ES, FR, AR
3. create a new post
4. insert a Tweet Action block
5. the block's placeholder text should be translated into the site language
7. view latest in ES, FR, AR
10. the labels should be translated into the site language (number of results, dropdowns, buttons)
